src/components/tutorials-landing-view/tutorials-landing-content-card-link/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useId } from '@react-aria/utils'
+import { IconArrowRight24 } from '@hashicorp/flight-icons/svg-react/arrow-right-24'
+import Badge from 'components/badge'
+import CardLink from 'components/card-link'
+import Tooltip from 'components/tooltip'
+import s from './tutorials-landing-content-card-link.module.css'
+
+type ContentCardLinkEyebrowPart = string
+
+interface ContentCardLinkBadge {
+	icon: $TSFixMe
+	label: string
+}
+
+interface ContentCardLinkProps {
+	backgroundImageUrl?: string
+	badges: ContentCardLinkBadge[]
+	description: string
+	eyebrowParts: ContentCardLinkEyebrowPart[]
+	headerImageUrl?: string
+	href: string
+	title: string
+}
+
+const TutorialsLandingContentCardLink = ({
```

Comment:
Oh one final thought; I don't think it's fully needed to prefix all of your components with `TutorialsLanding`; I think that is provided by its placement in the `tutorials-landing-view`!  I would personally like to avoid the added verbosity of that.  Not a super strong opinion!

Reply:
I normally do this name scoping/prefixing for a more efficient search and search+replace experience, but have removed!